everviz/config.py

```diff
@@ -16,14 +16,18 @@ def create_everviz_folder(everest_folder):
 
 
 def webviz_config(api):
-    create_everviz_folder(api.output_folder())
```

Comment:
you removed the `()` from `api.output_folder()` without updating the mock definition that is the reason the control test is failing.

Reply:
:+1:

Reply:
I would suggest keeping the `()` in place for now. I made the same update in my crossplot page PR

Reply:
But its a propery in the api, so it cannot have the `()`, right?

Reply:
that is true, but the API is mocked on the everviz side so changes in the everest API do not immediately reflect on the everviz side

Reply:
why not just follow the everest API, that is what is should be in the end, no?

Reply:
Yes, it should follow the API and it will follow the API. If you want to make the changes in your PR that is fine by me. You will just need to resolve more conflicts once the crossplots PR is merged or I will need to resolve more conflicts when this PR is merged. Anyway, the conflicts will be minor so it is up to you what you want to do.

Reply:
Sure, there are going to be some small conflicts in any case.